cl/lib/email_backends.py

```diff
@@ -45,6 +49,8 @@ def send_messages(
         base_backend = settings.BASE_BACKEND
         connection = get_connection(base_backend)
         connection.open()
+        r = make_redis_interface("CACHE")
+        check_or_create_email_sending_quota(r)
```

Comment:
Maybe a better name?

```suggestion
        inc_email_emergency_brake_or_raise(r)
```

Reply:
Sure!. Thanks for your comment.